### ISSUE TITLE
Cannot start BOINC Manager from commandline

### ISSUE THREAD

**Issue by romw** [BOINC/boinc#414]
**Reported by Ageless on 25 May 37775621 08:53 UTC**
Reported in [this thread](http://boinc.berkeley.edu/dev/forum_thread.php?id=2184) on the BOINC forums. 

By Francis:[getting errors when starting boinc-manager from the commandline. So I cant launch boinc.[[BR]([BR]]
Im)]

The error says :[xcb_xlib.c:42: xcb_xlib_lock: Controletest '!c->xlib.lock' faalt.[[BR]([BR]]
boincmgr:)]
Afgebroken[translation:[[BR]([BR]](English)]
boincmgr: xcb_xlib.c:42: xcb_xlib_lock: Controltest '!c->xlib.lock' failed.[someone give me a hint in solving this please ? I use Suse10.3

By Berend:[[BR]([BR]]
Aborted)[[BR]]

Can)]
Got the same problem and you're not the only one.[running OpenSuSE 10.3 on an Athlon X2 3800+ (64-bit) and an Intel T2400 duocore (32-bit).[[BR]([BR]]
I'm)]

I tried 4 different BOINC-clients and they all don't work. It seems there is a problem trying to connect to the localhost.

**Comment by romw** [comment]
**Commented by Ageless on 19 Sep 37775662 23:06 UTC**
Thanks for that, Nick. I must've not showered too well, overlooked that one. ;-)

In the mean time, more information on which versions can be affected. Berend posted back on my question what versions this is in:[I tried 5.4.11, 5.8.20 and 5.10.21 (Linux/X64).
With WINE I ran 5.10.20 (Win98+, Win2000+)
Only version 5.8.20 gave a GUI and running from a console gave the above mentioned error, the other Linux clients only ran from console but wouldn't start.

If running 5.8.20 (GUI) you get the message "disconnected from localhost". Trying to go o "Connect to computer.." freezes the GUI (In the Win-version it is possible to connect to the "Localhost".

The forum thread to the Fedora 8 beta version is: http://boinc.berkeley.edu/dev/forum_thread.php?id=2145
```

[http://boinc.berkeley.edu/dev/forum_thread.php?id=2145 http://boinc.berkeley.edu/dev/forum_thread.php?id=2145]([BR]]

```
Yes,)
